ISCSI/SCSI/SCSIReturnParameters/StandardInquiryData.cs

```diff
@@ -119,7 +119,7 @@ public byte[] GetBytes()
         {
             byte[] buffer = new byte[96];
             buffer[0] |= (byte)(PeripheralQualifier << 5);
-            buffer[0] |= (byte)(PeripheralQualifier & 0x1F);
+            buffer[0] |= (byte)((int)PeripheralDeviceType & 0x1F);

```

Comment:
This is a bugfix so it warrants a separate PR - I went ahead and pushed the fix to save you the trouble.